apps/libsignpost/signpost_api.h

```diff
@@ -261,23 +265,39 @@ int signpost_processing_reply(uint8_t src_addr, uint8_t message_type, uint8_t* r
 /* ENERGY API                                                             */
 /**************************************************************************/
 
+//message types
 enum energy_message_type {
     EnergyQueryMessage = 0,
     EnergyLevelWarning24hMessage = 1,
     EnergyLevelCritical24hMessage = 2,
     EnergyCurrentWarning60sMessage = 3,
+    EnergyReportMessage = 4,
```

Comment:
I think this name could be more specific. Something like `EnergyReportModuleConsumptionMessage`.